components/03-modules/products-grid-item/_products-grid-item.scss

```diff
@@ -0,0 +1,194 @@
+    overflow: hidden;
+
+    &__link {
+        &--name {
+            flex-basis: 70%;
+        }
+        @include links(active) {
+            display: block;
+            text-decoration: none;
+        }
+    }
+
+    &__content {
+        background: $products-grid-item__content-bg-color;
+        transition: all 0.5s ease-in-out;
+    }
+
+    &__details {
+        display: flex;
+        justify-content: space-between;
+        flex-wrap: wrap;
+    }
+
+    &__media {
+        display: flex;
+        flex-flow: row nowrap;
+        justify-content: center;
+        align-items: flex-start;
+        width: 100%;
+        margin-bottom: $products-grid-item__image-margin-bottom;
+    }
+
+    &__name {
+        font-weight: $products-grid-item__name-font-weight;
+        color: $products-grid-item__name-color;
+        font-size: $products-grid-item__font-size-medium;
+        padding: $products-grid-item__text-padding;
+
+        @include mq($screen-m) {
+            font-size: $products-grid-item__font-size-large;
+        }
+    }
+
+    &__button {
+        &--active {
+            @include mq($screen-l) {
+                display: inline-flex;
+            }
+        }
+    }
+
+    &__price {
+        flex: 1 0 auto;
+        font-weight: $font-weight-bold;
+        text-align: right;
+        color: $products-grid-item__price-base-color;
+        padding: $products-grid-item__text-padding;
+        margin-top: 6px; // product name aligment
+    }
+
+    &__old-price {
+        @include mq($screen-m) {
+            display: block;
+            margin-right: 0;
+        }
+    }
+
+    &__special-price {
+        @include mq($screen-m) {
+            display: block;
+        }
+    }
+
+    &__extra {
+        opacity: 0;
+        height: 0;
+        .products-grid-item--active & {
+            opacity: 1;
+            height: auto;
+        }
+    }
+
+    &__options,
+    &__details {
+        margin-bottom: $products-grid-item__base-spacing;
+    }
+
+    &__actions {
+        display: flex;
+        align-items: center;
+        justify-content: space-between;
+        width: 100%;
+    }
+
+    &__action-icons {
+        display: flex;
+        align-items: center;
+        justify-content: flex-end;
+    }
+
+    &__action-item {
+        box-shadow: none;
+        background: transparent;
+        border: none;
+        width: $products-grid-item__action-item-size;
+        height: $products-grid-item__action-item-size;
+        margin-right: $products-grid-item__action-item-spacing;
+        &:hover {
+            cursor: pointer;
+        }
+        &:last-child {
+            margin-right: 0;
+        }
+    }
+
+    &__action-icon {
+        width: $products-grid-item__action-icon-size;
+        height: $products-grid-item__action-icon-size;
+        &--wishlist {
+            width: $products-grid-item__action-icon-size--wishlist;
+            height: $products-grid-item__action-icon-size--wishlist;
```

Comment:
Let's export them from the zeplin as a 32x32px - ![](https://i.imgur.com/tqmtxXy.png)

Reply:
and replace heart.svg icon ?

Reply:
done